src/plugins/intel_cpu/src/nodes/rnn.cpp

```diff
@@ -514,6 +514,9 @@ void RNN::configurePortDataTypes() {
         // onednn doesn't have fp16 instance
         inDataTypes[xIdx] = outDataTypes[yIdx] = outDataTypes[hoIdx] = inDataTypes[hIdx] = memory::data_type::f32; // required by oneDNN.
 
+    if (cell_type == dnnl::algorithm::vanilla_augru && inDataTypes[aIdx] == memory::data_type::f16)
```

Comment:
1. Why we need this change? Is it oneDNN or ACL limitation? Lets add comment with explanation
2. Shouldn't this change be appllied for ARM arch only?
3. Do we have corresponding RNN test instances on ARM?

Reply:
1) OneDNN unsupported fp16 precision
2) For all devices where fp16 is used we will have exceptions
3) We don't have instances, but we need to create SLT separation first of all